src/contrib/cluster/Akka.Cluster.Sharding/ShardedDaemonProcess.cs

```diff
@@ -112,12 +112,23 @@ public DaemonMessageRouter(string[] entityIds, IActorRef shardingRef)
 
         protected override void OnReceive(object message)
         {
-            var nextId = _entityIds[_index % _entityIds.Length];
+            if (message is Broadcast broadcast)
+            {
+                var unwrapped = broadcast.Message;
+                foreach (var entityId in _entityIds)
+                {
+                    _shardingRef.Forward(new ShardingEnvelope(entityId, unwrapped));
+                }
+            }
```

Comment:
Special case, when message is wrapped inside the `Broadcast` envelope, send the payload to all worker entities.